Nodejs/Product/Nodejs/visualstudio_nodejs_repl.js

```diff
@@ -108,7 +108,9 @@ function processRequest(command) {
                 var obj;
                 // function f() { } should return undefined, wrapping it in parens
                 // produces a value, so ignore it if we have a function definition.
-                if (!/^\s*function/.test(command["code"])) {
```

Comment:
The old regexp would match any var starting with `function`. We only want to not apply wrapping for function declarations that use the function keyword explicitly. If I recall, function keyword in this context can either be  followed by either:

* Space and then identifier
* Paren for argument list (actually, this would make it an function expression but JS has a weird ambiguity about function declarations vs function expressions).
* A `*` for a generator.